profiles/coreos/amd64/sdk/make.defaults

```diff
@@ -1,3 +1,8 @@
 # Enable optimizations for common x86_64 CPUs
 CFLAGS="-O2 -pipe -mtune=generic"
 CXXFLAGS="${CFLAGS}"
+
+# Enable static-libs for bootstrapping b/c openssl needs static-libs on zlib
```

Comment:
I don't understand the comment about `static-libs` here.

Reply:
Oh, that's a left-over comment, sorry for that. I've updated the comment to describe why `cros_host` is there.